src/utils/globals.js

```diff
@@ -1,6 +1,6 @@
 /* global window, global, document */
-module.exports = {
```

Comment:
> why the code duplication between globals-internal.js and globals.js?

Let me double check - something here was causing problems with rollup I think. I'll add comments if I need to keep this change.